bin/reth/src/cli/ext.rs

```diff
@@ -56,23 +59,16 @@ pub trait RethNodeCommandConfig: fmt::Debug {
         &mut self,
         config: &Conf,
         components: &Reth,
-        registry: &mut RethModuleRegistry<
-            Reth::Provider,
-            Reth::Pool,
-            Reth::Network,
-            Reth::Tasks,
-            Reth::Events,
-        >,
-        modules: &mut TransportRpcModules,
+        node_components: &mut RethRpcComponents<'_, Reth>,
     ) -> eyre::Result<()>
     where
         Conf: RethRpcConfig,
         Reth: RethNodeComponents,
     {
         let _ = config;
         let _ = components;
-        let _ = registry;
-        let _ = modules;
+        let _ = node_components.registry;
+        let _ = node_components.modules;
```

Comment:
we only need this once
```suggestion
        let _ = node_components;
```